images/base/files/usr/local/bin/entrypoint

```diff
@@ -18,11 +18,119 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# If /proc/self/uid_map 4294967295 mappings, we are in the initial user namespace, i.e. the host.
+# Otherwise we are in a non-initial user namespace.
+# https://github.com/opencontainers/runc/blob/v1.0.0-rc92/libcontainer/system/linux.go#L109-L118
+userns=""
+if grep -Eqv "0[[:space:]]+0[[:space:]]+4294967295" /proc/self/uid_map; then
+  userns="1"
+  echo 'INFO: running in a user namespace (experimental)'
+fi
+
+validate_userns() {
+  if [[ -z "${userns}" ]]; then
+    return
+  fi
+
+  local nofile_hard
+  nofile_hard="$(ulimit -Hn)"
+  local nofile_hard_expected="64000"
+  if [[ "${nofile_hard}" -lt "${nofile_hard_expected}" ]]; then
+    # This ERROR can be demoted to WARNING when k/k PR gets merged: https://github.com/kubernetes/kubernetes/pull/92863
+    echo "ERROR: UserNS: expected RLIMIT_NOFILE to be at least ${nofile_hard_expected}, got ${nofile_hard}" >&2
+    exit 1
+  fi
+
+  local nf_conntrack_max
+  nf_conntrack_max="$(cat /proc/sys/net/netfilter/nf_conntrack_max)"
+  local nf_conntrack_max_expected="131072"
+  if [[ "${nf_conntrack_max}" != "${nf_conntrack_max_expected}" ]]; then
+    # This ERROR can be demoted to WARNING when k/k PR gets merged: https://github.com/kubernetes/kubernetes/pull/92863
+    echo "ERROR: UserNS: expected net.netfilter.nf_conntrack_max to be ${nf_conntrack_max_expected}, got ${nf_conntrack_max}" >&2
```

Comment:
I don't have the historical context, but I don't think this should be errors because users should be able to tweak this parameter, is not that something is going to break if this is not matching this specific value ... it can perform much worse, but that is another conversation

Reply:
Without this value, kube-proxy error out during attempting to set sysctl. So until 
 https://github.com/kubernetes/kubernetes/pull/92863 gets merged we need this check.

Reply:
ic, this is a configuration parameter to the kube-proxy, so this will ONLY work for the default value "131072", I don't think nobody will tweak this parameter and that you can get it beforehand so this seems the only way

Reply:
yeah, agree, I was curious only :)